cql3/column_identifier.cc

```diff
@@ -74,15 +74,6 @@ column_identifier_raw::prepare_column_identifier(const schema& schema) const {
         return ::make_shared<column_identifier>(_text, true);
     }
 
-    // We have a Thrift-created table with a non-text comparator.  We need to parse column names with the comparator
-    // to get the correct ByteBuffer representation.  However, this doesn't apply to key aliases, so we need to
-    // make a special check for those and treat them normally.  See CASSANDRA-8178.
-    auto text_bytes = to_bytes(_text);
-    auto def = schema.get_column_definition(text_bytes);
-    if (def) {
-        return ::make_shared<column_identifier>(std::move(text_bytes), _text);
-    }
```

Comment:
Point to think about:
If I understand correctly, this means that we're removing not only support for use of the Thrift protocol, but also for sstables created long ago with Thrift. Is this a correct observation? If it is, are we fine with that?

Reply:
i am not 100% sure. but since the thrift protocol has been deprecated from couple years, i doubt that there is any sstables around created with it. i think you are right. but i will keep this piece of code anyway just to be on the safe side.